discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/AddDocumentOptions.java

```diff
@@ -18,31 +18,33 @@
 import com.ibm.watson.developer_cloud.util.Validator;
 
 /**
- * the addDocument options.
+ * The addDocument options.
  */
 public class AddDocumentOptions extends GenericModel {
 
-  /** the ID of your environment. */
-  private String environmentId;
-  /** the ID of your collection. */
-  private String collectionId;
-  /**
-   * The ID of the configuration to use to process the document. If the `configuration` form part is also provided (both
-   * are present at the same time), then request will be rejected.
-   */
-  private String configurationId;
   /**
-   * The content of the document to ingest.The maximum supported file size is 50 megabytes. Files larger than 50
-   * megabytes is rejected.
+   * The media type of file.
    */
+  public interface FileMediaType {
+    /** application/json. */
+    String APPLICATION_JSON = "application/json";
+    /** application/msword. */
+    String APPLICATION_MSWORD = "application/msword";
+    /** application/vnd.openxmlformats-officedocument.wordprocessingml.document. */
+    String APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_WORDPROCESSINGML_DOCUMENT
+        = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
+    /** application/pdf. */
+    String APPLICATION_PDF = "application/pdf";
+    /** text/html. */
+    String TEXT_HTML = "text/html";
+    /** application/xhtml+xml. */
+    String APPLICATION_XHTML_XML = "application/xhtml+xml";
+  }
```

Comment:
FYI ... the FileMediaType interface will go away with the changes I am making for file handling.